src/ActionModal/components/DelegationModal.vue

```diff
@@ -89,6 +89,11 @@
           @click.native="setMaxAmount()"
         />
       </TmFieldGroup>
+      <span v-if="network" class="form-message">
+        In Polkadot staked tokens are independent from picked validators. Want
```

Comment:
do we need this message?

Reply:
well the UX is different from Cosmos, so I wanted to make this clearer.

Reply:
this message should be moved up to line 17